### DISCUSSION TITLE
Can we connect web browsing to prompt template?

### DISCUSSION THREAD

**Discussion by Krataus** [FlowiseAI/Flowise#593] [Q&A] [answered]
I would like to be able to use the prompt template to initiate the search on web browsing and have a multitude of steps to get information from the results generated from the web browsing. to dissect, and organise informaation etc... 
Thank you for your time. ( in flowise)

**Comment by HenryHengZJ** [comment, marked as answer]
I think you'll have to play around with the prompt to achieve that. I'd say try it on chatgpt and if that works, you should be able to do it on Flowise too

**Reply by Krataus** [reply]
Thank you
